cockatrice/src/logger.h

```diff
@@ -0,0 +1,37 @@
+#ifndef LOGGER_H
+#define LOGGER_H
+
+#include <QTextStream>
+#include <QFile>
+#include <QVector>
+#include <QString>
+
+class Logger : public QObject {
+Q_OBJECT
+public:
+    static Logger& getInstance()
+    {
+        static Logger instance;
+        return instance;
+    }
+private:
+    Logger();
+    ~Logger();
+    // Don't implement
```

Comment:
I'll add a more descriptive comment: `// Singleton - Don't implement copy constructor and assign operator`